tests/tests/control_stochy

```diff
@@ -33,11 +33,12 @@ export IAER=5111
 export DO_SPPT=.T.
 export DO_SHUM=.T.
 export DO_SKEB=.T.
+export RESTART_INTERVAL=6
 export SKEB=0.3
 export SHUM=0.003
 export SPPT=0.2
 
 
-export FV3_RUN=control_run.IN
+export FV3_RUN=control_stochy_run.IN
```

Comment:
It seems to me the only difference between the control_run.IN and control_stochy_run.IN is that the stochy is using 6hr restart files while control is using 12, maybe in the future we can add a restart file time to the script and unify the two scripts so that we don't need to maintain two scripts.

Reply:
Yes, that can be added so we don't have 2 scripts.